src/ussd_public.js

```diff
@@ -581,6 +619,66 @@ go.app = function() {
             });
         });
 
+        self.add('state_optout_all', function(name) {
+            if (self.im.user.answers.household_id === null) {
```

Comment:
I explicitly set it to `null` in `state_check_receiver_role`. If it is `undefined` something would have gone wrong and I don't want to try acting on that.